### ISSUE TITLE
how to show notebooks by tree view in IOS  ?

### ISSUE THREAD

**Issue by MatthewLee123** [HappenApps/Quiver#1265]
in Mac, all notebooks are organised as tree view, but in iPhone, it's so massy. 
can you make them have the same vision ?

**Comment by MatthewLee123** [comment]
@ylian

**Comment by ylian** [comment]
@MatthewLee123 Yeah, will be fixed in the next update.

**Comment by MatthewLee123** [comment]
thanks. can you please let me know when approximately the next update will be published ?

**Comment by ylian** [comment]
Can't give a definite date. I am working on several things at the moment and will push them out as they are ready.

**Comment by MatthewLee123** [comment]
Hi, ylian.

it's been a long time. just wanna know how this update is going ?

**Comment by ylian** [comment]
Almost there. Expect iOS 1.1 this month. It's a complete rewrite in Swift.

**Comment by MatthewLee123** [comment]
@ylian Hi mate, is iOS1.1 published ?

**Comment by ylian** [comment]
No, not yet. The rewrite is completed, but not stable enough yet.
